src/commands/createAccountsService.js

```
@@ -0,0 +1,74 @@
+        environmentConfig.accountName
+      );
+      if (accountId) {
+        logger.info('Waiting 30 seconds for account to be ready before creating IAM user...');
+        await new Promise(resolve => setTimeout(resolve, 30000));
+        await createIAMUser(accountId, options.username);
+      }
+    }
+  }
+
+  async handleExistingUser(accountId, username) {
+    logger.info(`Retrieving credentials for existing user...`);
```

ESLint

[error] 'logger' is not defined.

(no-undef)